docs/dev/Lua API.rst

```diff
@@ -1679,21 +1695,45 @@ Units module
 
   Finds (or creates if requested) a misc trait object with the given id.
 
+* ``dfhack.units.getRaceNameById(race_id)``
+* ``dfhack.units.getRaceName(unit)``
+
+  Get raw token name (e.g., "DWARF").
+
+* ``dfhack.units.getRaceReadableNameById(race_id)``
+* ``dfhack.units.getRaceReadableName(unit)``
+* ``dfhack.units.getRaceNamePluralById(race_id)``
+* ``dfhack.units.getRaceNamePlural(unit)``
+
+  Get human-readable name (e.g., "dwarf" or "dwarves").
+
+* ``dfhack.units.getRaceBabyNameById(race_id[,plural])``
+* ``dfhack.units.getRaceBabyName(unit[,plural])``
+* ``dfhack.units.getRaceChildNameById(race_id[,plural])``
+* ``dfhack.units.getRaceChildName(unit[,plural])``
+
+  Get human-readable baby or child name (e.g., "dwarven baby" or "dwarven child").
+
+* ``dfhack.units.getReadableName(unit)``
+
+  Returns a string that includes the language name of the unit (if any), the
+  race of the unit (if different from fort), whether it is trained for war or
+  hunting, any syndrome-given descriptions (such as "necromancer"), the training
+  level (if tame), and profession or noble role.
+
+* ``dfhack.units.getPhysicalDescription(unit)``
+
+  Should return physical description of the unit, but may not work for a given
+  DFHack build.
```

Comment:
looking at the code, this might never work for any DFHack build and should probably be removed.

Does it produce any output on Windows? I can confirm it does nothing on Linux.

Reply:
Nope. I suppose it worked for [v47.04](https://github.com/DFHack/df-structures/blob/dd8b9335007941c3aea27474b0a08ba31f6bc97f/symbols.xml#L3408), but the address of the function was never maintained.